volontulo/templates/base.html

```diff
@@ -18,6 +18,7 @@
             {% endblock %}
         </div>
 
+        <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
```

Comment:
Please insert this into block
{% block scripts %}
<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
{% endblock %}